src/frontend/komponenter/Fagsak/OpprettFagsak.tsx

```diff
@@ -1,14 +1,45 @@
-import { Lukknapp } from 'nav-frontend-ikonknapper';
+import { Lukknapp, Nesteknapp } from 'nav-frontend-ikonknapper';
 import { Knapp } from 'nav-frontend-knapper';
 import { Input, Select } from 'nav-frontend-skjema';
 import { Systemtittel, Undertittel } from 'nav-frontend-typografi';
 import * as React from 'react';
 import { behandlingstyper } from '../../typer/fagsak';
+import { sjekkeFødselsnummer, ErrorId } from './DataValidator';
 
 const OpprettFagsak: React.FunctionComponent = () => {
     const [søkersFødselsnummer, settSøkersFødselsnummer] = React.useState('');
     const [barnasFødselsnummer, settBarnasFødselsnummer] = React.useState<string[]>(['']);
 
+    const validereFødselsnummer = (): boolean => {
```

Comment:
When introducing IFelt we can generalize this function and move it to a util file of some kind.